cmds/core/dd/dd.go

```diff
@@ -51,17 +51,21 @@ import (
 	"github.com/u-root/u-root/pkg/progress"
 )
 
+type command struct {
```

Comment:
this structure is unused, probably can be removed or add run to it

Reply:
yes, though, sooner or later, we'll want it back :-)